src/inverseRobot.js

```diff
@@ -7,7 +7,32 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  const equalValue = Object.entries(robot).reduce((acc, [ k, v ]) => {
+    if (!acc[v]) {
+      acc[v] = [];
+    }
+
+    acc[v].push(k);
+
+    return acc;
+  },
+  {});
+
+  for (const value in equalValue) {
+    const lenValue = equalValue[value].length;
+
+    if (lenValue >= 2) {
+      return null;
+    }
+  }
+
+  for (const [key, value] of Object.entries(robot)) {
+    invertedRobot[value] = key;
+  }
```

Comment:
You can do this task with this cycle only. You can check if `invertedRobot` already has a current key (using `invertedRobot.hasOwnProperty(value)`) and return `null` for this case.